### ISSUE TITLE
Automatically send verification email after registration

### ISSUE THREAD

**Issue by peteole** [ory/kratos#2392]
### Preflight checklist

- [X] I could not find a solution in the existing issues, docs, nor discussions.
- [X] I agree to follow this project's [Code of Conduct](https://github.com/ory/kratos/blob/master/CODE_OF_CONDUCT.md).
- [X] I have read and am following this repository's [Contribution Guidelines](https://github.com/ory/kratos/blob/master/CONTRIBUTING.md).
- [ ] This issue affects my [Ory Cloud](https://www.ory.sh/) project.
- [X] I have joined the [Ory Community Slack](https://slack.ory.sh).
- [ ] I am signed up to the [Ory Security Patch Newsletter](https://ory.us10.list-manage.com/subscribe?u=ffb1a878e4ec6c0ed312a3480&id=f605a41b53).

### Describe your problem

I am currently trying to set up address verification. As I understood the docs the intended process looks as follows:

1. User starts registration flow
2. Kratos responds with flow ID
3. User submits personal data referencing flow ID
4. Kratos confirms registration, optionally setting a session cookie using the session hook and redirecting to another page
5. User starts new verification flow
6. Kratos responds with verification flow ID
7. User submits their email referencing verification flow ID
8. Kratos confirms verification and sends email
9. User clicks email link, sending a token to kratos with browser
10. Kratos sets user as verified, optionally setting session cookie using session hook and redirecting to dashboard page

To me it seems this process has too many steps that can go wrong. I do not want to bother my users with the need to manually start a verification flow.

### Describe your ideal solution

There is an option to make kratos automatically send the verification email after registration, probably using a hook. Maybe this is even possible today using some webhook magic?

### Workarounds or alternatives

Make users verify their email with an UI element or write custom code starting the verification after registration. This is annoying since either it's done in the UI itself which forces you to handle all kinds of errors such as the user closing the website in the middle of the process or it's done in the backend which forces another coupling of kratos and the rest of the backend since it needs to listen to registrations.

### Version

helm chart: 0-21.7
kratos:0.8.0-alpha3

**Comment by aeneasr** [comment]
Hey, I think there's a misunderstanding. If enabled, and configured correctly, verification emails get sent out directly after registreation. There is no need for the user to kick off the process manually. See: https://www.ory.sh/docs/kratos/guides/account-activation-email-verification

Closing this issue as it appears to be a misconfig / misunderstanding. Please feel free though to comment if you face issues, someone from devrel (e.g. @vinckr or @gen1us2k will be happy to assist!)

**Comment by peteole** [comment]
Ok thanks, reading through the docs once more carefully, you're right, I found something saying that verification mails can be sent out after registration. However to me it is not totally clear from the docs that the verification mail is sent _automatically_ during registration since the there's only a brief mention of _sending_ the mail after registration and the rest of the article talks about _manually_ requesting registration using a flow. Would be nice to have a section that points out how the automatic sending of verification emails behaves and how it can be configured.

By the way there's another point I stumbled over: The docs for verification and registration both stop at the point where the email is sent. This confuses me since the user clicking the link and kratos redirecting to a page, setting a session token seems like an essential part of the process.
